docs/source/nlp/punctuation_and_capitalization.rst

```diff
@@ -0,0 +1,345 @@
+
+.. code::
+
+   .
+   |--output_dir
+     |-- labels_dev.txt
+     |-- labels_train.txt
+     |-- text_dev.txt
+     |-- text_train.txt
+
+Training Punctuation and Capitalization Model
+---------------------------------------------
+
+In the Punctuation and Capitalization Model, we are jointly training two token-level classifiers on top of a pre-trained \
+language model, such as `BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding <https://arxiv.org/abs/1810.04805>`__ :cite:`nlp-punct-devlin2018bert`.
+Unless the user provides a pre-trained checkpoint for the language model, the language model is initialized with the
+pre-trained model from `HuggingFace Transformers <https://github.com/huggingface/transformers>`__.
+Example of model configuration file for training the model could be found at: `NeMo/examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml>`__.
+
+The specification can be roughly grouped into the following categories:
+
+* Parameters that describe the training process: **trainer**
+* Parameters that describe the datasets: **model.dataset**, **model.train_ds**, **model.validation_ds**
+* Parameters that describe the model: **model**
+
+More details about parameters in the config file could be found below and in the `model's config file <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml>`__:
+
+
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| **Parameter**                             | **Data Type**   |  **Description**                                                                                             |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| pretrained_model                          | string          | Path to the pre-trained model .nemo file or pre-trained model name                                           |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| model.dataset.data_dir                    | string          | Path to the data converted to the specified above format                                                     |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| trainer.max_epochs                        | integer         | Maximum number of epochs to train the model                                                                  |
```

Comment:
paramaters of trainer should be in nemo core doc and just give a ref to there? @ericharper